modules/lang-painless/src/main/java/org/elasticsearch/painless/lookup/PainlessLookupBuilder.java

```diff
@@ -126,14 +126,55 @@ public int hashCode() {
     private static final Pattern METHOD_NAME_PATTERN = Pattern.compile("^[_a-zA-Z][_a-zA-Z0-9]*$");
     private static final Pattern FIELD_NAME_PATTERN  = Pattern.compile("^[_a-zA-Z][_a-zA-Z0-9]*$");
 
-    private final List<Whitelist> whitelists;
+    public static PainlessLookup whitelistsToPainlessLookup(List<Whitelist> whitelists) {
```

Comment:
I would name this more simply like `buildFromWhitelist`.

Reply:
Done.